templates/cluster-tsp/main.go

```diff
@@ -119,7 +119,7 @@ func clusterSolution(input input) (kmeans.Solution, error) {
 	}
 
 	for idx := 0; idx < len(input.Vehicles); idx++ {
-		maximumPoints[idx] = len(input.Stops) / len(input.Vehicles)
+		maximumPoints[idx] = (len(input.Stops) / len(input.Vehicles)) + 1
```

Comment:
What is the logic behind `len(input.Stops) / len(input.Vehicles)`? Why does the average number of stops per vehicle equals the maximum points?

I am asking because I am wondering if `s/v + 1` is right here or rather `ceil(float64(s)/float64(v))`?

Reply:
no real requirement. it is mainly to avoid an input problem that could happen in the very rare case of 1s and 2v. We can also use ceil to avoid this problem, I have no opinion.

Reply:
Ah ok, `maximumPoints` is the upper bound on points per cluster. You could also do `(s+v - 1) / v` which is equivalent to `ceil(float64(s)/float64(v))` for integer arithmetic. However you need to ensure that `v > 0` (was the case before, otherwise there is a panic).

Also we can move the computation outside of the loop.

Reply:
That being said, for `v > 0` and `math.Ceil(float64(len(...))/float64(len(...)))` may be more readable.

Reply:
Did you reach a consensus here, or is another opinion required? :sweat_smile:

Reply:
I think I lean towards `Ceil` too, but have no strong opinion. Will rubber stamp to potentially unblock.